data/games/under_night_in_birth_exe_late_st/movespecs/wagner.json

```diff
@@ -0,0 +1,150 @@
+        ],
+        [
+            {
+                "name": "Schild",
+                "inputs": [
+                    "B+C"
+                ]
+            },
+            {
+                "name": "Schild (charged)",
+                "inputs": [
+                    "[B+C]"
+                ]
+            },
+            {
+                "name": "Throw",
+                "inputs": [
+                    "A+D"
+                ]
+            },
+            {
+                "name": "Guard Thrust",
+                "inputs": [
+                    "214D"
+                ]
+            },
+            {
+                "name": "Veil Off",
+                "inputs": [
+                    "A+B+C"
+                ]
+            }
+        ],
+        [
+            {
+                "name": "Kugel Blitz",
+                "inputs": [
+                    "236A",
+                    "236B",
+                    "236C"
+                ]
+            },
+            {
+                "name": "Wackenroder",
+                "inputs": [
+                    "623A",
+                    "623B",
+                    "623C"
+                ]
+            },
+            {
+                "name": "Sturm Brecher",
+                "inputs": [
+                    "214A",
+                    "214B",
+                    "214C"
+                ]
+            },
+            {
+                "name": "Sturm Brecher (charged)",
+                "inputs": [
+                    "214[A]",
+                    "214[B]",
+                    "214[C]"
+                ]
+            },
+            {
+                "name": "Sturm Brecher (with buff)",
+                "inputs": [
+                    "214A",
+                    "214B",
+                    "214C"
+                ],
+                "conditions": "after Lohe Garde"
+            },
+            {
+                "name": "Sturm Brecher (charged) (with buff)",
+                "inputs": [
+                    "214[A]",
+                    "214[B]",
+                    "214[C]"
+                ],
+                "conditions": "after Lohe Garde"
+            },
+            {
+                "name": "Lohen Schneide / Lohe Garde",
+                "inputs": [
+                    "22A",
+                    "22B",
+                    "22C"
+                ]
+            },
+            {
+                "name": "Sturm Schlag",
+                "inputs": [
+                    "j.236A",
+                    "j.236B",
+                    "j.236C"
+                ]
+            },
+            {
+                "name": "Sturm Schlag (charged)",
+                "inputs": [
+                    "j.236[A]",
+                    "j.236[B]",
+                    "j.236[C]"
+                ]
+            }
+        ],
+        [
+            {
+                "name": "Hitze Falke",
+                "inputs": [
+                    "41236D"
+                ]
+            },
```

Comment:
I believe the `Hitze Falke` and `Megiddo Ausserst Grousse` objects each need to be into their own arrays, otherwise they'll both be categorized under `Infinite Worth` rather than the first in `Infinite Worth` and the second in `Infinite Worth EXS`.